pkg/tftp/tftp.go

```diff
@@ -319,19 +320,27 @@ func executeGet(client ClientIf, host, port string, files []string) error {
 			}
 		}
 
-		nR := 0
 		data := make([]byte, 0)
 		for {
-			fmt.Println("loop")
-			readData := make([]byte, 10)
-			rD, err := resp.Read(data)
+			// Transmission with less than 512 byte indicates termination of connection
+			// See: https://datatracker.ietf.org/doc/html/rfc1350#section-2
+			readData := make([]byte, 512)
```

Comment:
I think you don't need to allocate `readData` in every iteration and  copy n bytes to `data`?

Reply:
Found a better solution. Thank you for the review :)